build-tools/packages/build-cli/src/lib/git.ts

```diff
@@ -2,12 +2,17 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+import { Context, Package } from "@fluidframework/build-tools";
+import path from "node:path";
+import readPkgUp from "read-pkg-up";
 import { SimpleGit, SimpleGitOptions, simpleGit } from "simple-git";
-// flase positive?
+
+// false positive?
```

Comment:
nits: comment can be removed?

Reply:
I expanded it to explain that the eslint disable shouldn't be needed but is for some reason.